### PR TITLE
Add anchor tag to documentation inner title

### PR DESCRIPTION
**Pull request by service-paradis**
### Proposed changes
- Add anchor tag to documentation inner title
- Add management for inner links

captured

Easier to create link to direct section of the documentation.

This is a proposal. Could be discussed.

### CODE REVIEW

**Comment by jtommy** [comment]
You're telepatich.. I was thinking about it.
It's a great feature!

**Comment by service-paradis** [comment]
haha, glad it helps!

File: docs/templates/Documentation.vue [hunk 1]
```diff
@@ -51,11 +51,17 @@ export default {
...
+        },
+        scrollTo(hash) {
+            location.href = hash
         }
     },
     mounted() {
         this.$eventHub.$on('navigate', this.setMeta)
         this.setMeta(this.$router.currentRoute.meta)
+        if (this.$route.hash) {
+            setTimeout(() => this.scrollTo(this.$route.hash), 1)
```
Comment (service-paradis) [inline docs/templates/Documentation.vue]: @jtommy 
I found that without a timeout, direct link access (with an anchor) is not always working.

Comment (jtommy) [reply docs/templates/Documentation.vue]: Mmm I'll try using next tick

**Pull request merged**